src/pages/query-view/query-view.js

```diff
@@ -4,6 +4,7 @@ import React from "react";
 import DatasetQuery from "../../components/dataset-query";
 import ReactTable from "react-table";
 import LoadingElement from "../../components/generic-elements/loading-element";
+import _ from 'lodash'
```

Comment:
Alternatively, if you don't want to have to carry the `_` around, I like to just import the functions we need

Reply:
We're importing isNull and isNaN so I figured it was better to call out that they were not the builtins.